Cargo.lock

```diff
@@ -5715,15 +5805,52 @@ dependencies = [
 
 [[package]]
 name = "time"
-version = "0.1.42"
+version = "0.1.43"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "db8dcfca086c1143c9270ac42a2bbd8a7ee477b78ac8e45b19abfb0cbede4b6f"
+checksum = "ca8a50ef2360fbd1eeb0ecd46795a87a19024eb4b53c5dc916ca1fd95fe62438"
 dependencies = [
  "libc",
- "redox_syscall",
  "winapi",
 ]
 
+[[package]]
+name = "time"
+version = "0.2.10"
```

Comment:
As a note: this will introduce duplicated `time`

Reply:
It probably doesn't compile anyway. That update required code changes here: https://github.com/mvdnes/zip-rs/pull/125/files Probably requires the same kind of changes here too.